flink-autoscaler/pom.xml

```diff
@@ -66,6 +66,12 @@ under the License.
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz</artifactId>
+            <version>2.3.2</version>
```

Comment:
Would you mind extract a property for this version? Such as : `quartz.version`.

Reply:
sure, already extracted